hyperspy/tests/drawing/test_plot_signal1d.py

```diff
@@ -87,7 +92,7 @@ def setup_teardown(request, scope="class"):
 @pytest.mark.usefixtures("setup_teardown")
 class TestPlotSpectra():
 
-    s = hs.signals.Signal1D(scipy.misc.ascent()[100:160:10])
+    s = hs.signals.Signal1D(ascent()[100:160:10])
```

Comment:
Does it make sense to use `skimage.data.camera` here as in the docstrings?

Reply:
Since the test uses image comparison, we need to keep using `ascent` or we need to update the baseline images. I went for the first option because I find it more simple.